impeller/renderer/command_buffer.h

```diff
@@ -65,6 +65,11 @@ class CommandBuffer {
 
   [[nodiscard]] bool SubmitCommands();
 
+  //----------------------------------------------------------------------------
+  /// @brief      Force execution of pending GPU commands.
+  ///
+  void Flush();
```

Comment:
We should use the more descriptive names for these in the modern backends. `WaitUntilScheduled` and `WaitUntilSubmitted`. This one is a wait for scheduled.

Reply:
done